operations/template/alert.tf

```
@@ -0,0 +1,43 @@
+resource "azurerm_monitor_action_group" "monitor" {
```

Consider something more descriptive than `monitor`.  Perhaps `notify_slack_email`.
```suggestion
resource "azurerm_monitor_action_group" "notify_slack_email" {
```